instrumentation/build.gradle.kts

```diff
@@ -58,11 +58,12 @@ android {
 }
 
 dependencies {
-    implementation("androidx.appcompat:appcompat:1.6.1")
-    implementation("androidx.core:core:1.12.0")
-    implementation("androidx.navigation:navigation-fragment:2.7.3")
+    implementation(platform(project(":dependencyManagement")))
```

Comment:
I'm not sure if it's going to work, but I was thinking that maybe we can just add this dependency into the java-library, android-app, and android-library convention files and maybe that'd help to not having to add it in each subproject like in here.

Reply:
If it works that would be cool. I'll give that a shot.